src/Aspire.Hosting.Azure.Kusto/Aspire.Hosting.Azure.Kusto.csproj

```diff
@@ -4,8 +4,8 @@
     <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
     <IsPackable>true</IsPackable>
     <EnablePackageValidation>false</EnablePackageValidation> <!-- TODO: Remove this once the package ships -->
-    <PackageTags>aspire integration hosting kusto</PackageTags>
-    <Description>Kusto support for .NET Aspire.</Description>
+    <PackageTags>aspire integration hosting azure kusto</PackageTags>
+    <Description>Azure Kusto support for .NET Aspire.</Description>
```

Comment:
🤔 I wonder whether we should be calling this `Aspire.Hosting.Azure.DataExplorer` inline with the public branding. Maybe the Kusto team will answer this for us.

Reply:
I'll ultimately defer to the team, but today I don't see any official packages using the DataExplorer name. Everything is under the Kusto name: https://www.nuget.org/packages?q=Kusto&includeComputedFrameworks=true&prerel=true

Reply:
Generally the way we name these things is that for Azure-based resources we align the names to the matching `Azure.Provisioning.XYZ` package. The name of that package is mostly informed by the name of the resource provider. 

In this case the name of the resource provider is Kusto I believe so I think we have the right name. I'll resolve this comment.